cni/network/multitenancy_test.go

```diff
@@ -50,30 +70,68 @@ type cnsAPIName string
 
 const (
 	GetAllNetworkContainers cnsAPIName = "GetAllNetworkContainers"
+	RequestIPs              cnsAPIName = "RequestIPs"
+	ReleaseIPs              cnsAPIName = "ReleaseIPs"
 )
 
 var (
 	errUnsupportedAPI             = errors.New("Unsupported API")
+	errNoRequestIPFound           = errors.New("No Request IP Found")
```

Comment:
what do these mean?

Reply:
i dont think we are testing actual code paths but creating these mock errors and returning from that in mock function. @paulyufan is that right?